### ISSUE TITLE
Not all prior art is indexed and searchable

### ISSUE THREAD

**Issue by metasj** [prior-art-archive/overview#2]
We should find and index all the art, or pointers to collections of the art, or lists of entities maintaining pointers to collections of the art.

**Comment by slifty** [comment]
Assuming this doesn't mean "all prior art in the universe" and instead are talking about prior art that has been uploaded but isn't indexed.  That said, I kind of hope that this IS an issue that will stay open until we have properly indexed all prior art in human history.

What is the nature of the prior art that isn't indexed?  Is there a pathway that doesn't lead to indexing automatically?

**Comment by joeltg** [comment]
@metasj what do you mean?

**Comment by slifty** [comment]
I AM STARTING TO THINK THAT THIS ISSUE WAS ONE FOR THE LULZ.

(we should probably mark it as complete with an appropriately lulz-inspiring tag)

**Comment by metasj** [comment]
Yes, your second guess was right.  All of the art (though we have to scope that; it's something structured as art, not just all documents.

**Comment by slifty** [comment]
DARN!  Well then we'll have to find a new, better use of this tag.

@metasj and @joeltg would either of you have insight around the question of `What is the nature of the prior art that isn't indexed? Is there a pathway that doesn't lead to indexing automatically?`

**Comment by metasj** [comment]
In all seriousness:  the founding premise of the archive is that prior art should be [as discoverable as patent literature], particularly to the extent examiners and inventors would find useful. 

This could be rescoped / subdivided into

- what prior art v0 should contain
- possible art for v1, v2
- other art that might be useful to the current audience 
- additional efforts to trace the arc of ideas through recorded knowledge
